lib/internal/heap_utils.js

```diff
@@ -1,16 +1,37 @@
 'use strict';
 const {
-  Symbol
+  Symbol,
+  Uint8Array,
 } = primordials;
 const {
   kUpdateTimer,
   onStreamRead,
 } = require('internal/stream_base_commons');
 const { owner_symbol } = require('internal/async_hooks').symbols;
 const { Readable } = require('stream');
+const { validateObject, validateBoolean } = require('internal/validators');
 
 const kHandle = Symbol('kHandle');
 
+function getHeapSnapshotOptions(options = {
+  exposeInternals: false,
+  exposeNumericValues: false
+}) {
+  validateObject(options, 'options');
```

Comment:
No need to initialize an object here I think.
```suggestion
function getHeapSnapshotOptions(options = kEmptyObject) {
  validateObject(options, 'options');
```